content/influxdb/v2.0/reference/rc0-upgrade-guide.md

```diff
@@ -0,0 +1,294 @@
+
+### Why is this manual process required?
+
+To ensure that existing InfluxQL integrations work with the latest release,
+we had to make breaking changes to the underlying storage engine for InfluxDB 2.0.
+
+{{% note %}}
+If you have questions upgrading please
+[open an issue](https://github.com/influxdata/influxdb/issues)
+or join the [Community Slack workspace](https://influxcommunity.slack.com/) to get immediate help.
+{{% /note %}}
+
+## 1. Disable existing integrations
+
+To begin, shut off all integrations that are reading, writing, or monitoring your InfluxDB instance.
+This includes Telegraf, client libraries, and any custom applications.
+
+## 2. Stop existing InfluxDB beta instance
+
+Next, shut down your existing InfluxDB beta instance.
+You can manually kill the individual process using **Control+c**
+(or by finding the process ID with `ps aux | grep -i influxd` and using `sudo kill -9 <PID>`).
+If you've set `influxd` to run as a system process, follow the same steps you would use to disable any system process.
+
+## 3. (Optional) Rename existing InfluxDB binaries
+
+To easily identify your existing InfluxDB binaries, rename them `influx_old` and `influxd_old`.
+This is helpful if you've installed the binaries in your path.
+We use the names `influxd_old` for this guide, but you can use whatever you like.
+
+## 4. Move existing data and start the latest InfluxDB
+
+If you have not already, [download the InfluxDB OSS 2.0rc0](https://portal.influxdata.com/downloads/).
+Download and follow the install instructions for `influxd` and `influx`.
+Be careful not to overwrite the existing binaries(that they may or may not have renamed earlier).
+
+To move data between the two instances, first configure both the old and new instances of InfluxDB to run at the same time.
+If you download the latest InfluxDB beta and try to start it with existing data,
+most likely it won't start and you'll see the following error message in your terminal:
+
+```
+Incompatible InfluxDB 2.0 version found.
+Move all files outside of engine_path before influxd will start.
+```
+
+To ensure you don't get this message, run the following command to move your existing data to a another location (anywhere you like):
+
+```sh
+mv ~/.influxdbv2 ~/.influxdbv2_old
+```
+
+(You can move it to wherever you'd like.)
+When we start the old instance again, we will tell it where your data files are located.
+
+Start the latest InfluxDB version by running
+
+```sh
+influxd
+```
+
+{{% note %}}
+If you were using specific [command line flags](/influxdb/v2.0/reference/cli/influxd/#flags) for InfluxDB beta, you can use those same command line flags.
+{{% /note %}}
+
+Since the data folder has been moved, everything will be empty.
+You can visit http://localhost:8086 in your browser and see a setup page, but don't go through the setup process yet.
```

Comment:
Why are we telling them this?

Reply:
Hm... not sure. Maybe going through the setup process at this point could be damaging in some way? Should check one this...